src/layers/conv.jl

```diff
@@ -118,29 +121,27 @@ function (l::GCNConv)(g::GNNGraph,
         end
     end
     Dout, Din = size(l.weight)
-    if Dout < Din
-        # multiply before convolution if it is more convenient, otherwise multiply after
-        x = l.weight * x
-    end
```

Comment:
Why this has been removed?

Reply:
if the multiplication happens before convolution, I used to have an error about dimensions or `MethodError: no method matching *(::Matrix{Float32}, ::Tuple{Matrix{Float32}, Matrix{Float32}})`
as the comment says `if it is more convenient`, I understood these lines are not a must and cut them out

Reply:
We do want that performance optimization

Reply:
it's either this optimization and `DimensionMismatch: arrays could not be broadcast to a common size; got a dimension with lengths 4 and 2` error message (so no support for GCNConv layer) or no optimization and no error message.
if there is third option, I'm willing to implement it, but need a hint what's that. I tried to get rid of that error for long already